### ISSUE TITLE
[Bug]: Undefined array key with FreeBSD + jails

### ISSUE THREAD

**Issue by cyrilst** [nextcloud/serverinfo#360]

**Comment by KDederichs** [comment]
Actually I think that's related to #270 and #358 . I just openend a pull request for those as well :) (Waiting for review)

**Comment by tflidd** [comment]
You mean this PR https://github.com/nextcloud/serverinfo/pull/372?

**Comment by KDederichs** [comment]
Yeah that one/the backport PR's that come out of it

**Comment by tflidd** [comment]
so once the backport https://github.com/nextcloud/serverinfo/pull/373 is merged, we can close this.

**Comment by KDederichs** [comment]
Yeah, I mean technically it's still missing the packport PR's for 23 and 22 but I have no clue how that backport bot works :D

**Comment by tflidd** [comment]
under a pull request you can just add `/backport to stable24` and the bot will try to backport.

**Comment by KDederichs** [comment]
Yeah, there's backport comments but it only created the one for 24, the one for 23 and 22 seem to be missing. Maybe they get queued up will 24 is merged?

**Comment by joshtrichards** [comment]
Looks like the backports got sorted out on the PRs. So this seems like it should be closed now, correct?

I can't test as I'm not using FreeBSD, but this looks to be in all currently supported releases (and then some at this point).

**Comment by cyrilst** [comment]
Correct. I successfully tested 26.0.2 with FreeBSD + jails.
Thanks for closing the issue.
